kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
```

Comment:
I wonder if it wouldn't be better to move all `from kivy.config import Config` in that .py file to the top (there are 3).